execution/executor-benchmark/src/lib.rs

```diff
@@ -146,6 +146,27 @@ pub fn run_benchmark<V>(
 
     let mut start_time = Instant::now();
     let start_gas = TXN_GAS_USAGE.get_sample_sum();
+
+    let start_execution_total = APTOS_EXECUTOR_EXECUTE_BLOCK_SECONDS.get_sample_sum();
+    let start_vm_only = APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS.get_sample_sum();
+    let other_labels = vec![
+        ("1.", true, "verified_state_view"),
+        ("2.", true, "apply_to_ledger"),
+        ("2.1.", false, "sort_transactions"),
+        ("2.2.", false, "calculate_for_transaction_block"),
+        ("2.2.1.", false, "produce_sharded_state_cache"),
+        ("2.2.2.", false, "calculate_block_state_updates"),
+        ("2.2.3.", false, "calculate_usage"),
+        ("2.2.4.", false, "make_checkpoint"),
+        ("2.3.", false, "assemble_ledger_diff_for_block"),
+        ("2.3.1.", false, "calculate_events_and_writeset_hashes"),
+        ("3.", true, "as_state_compute_result"),
+        ("4.", true, "get_txns_to_commit"),
+    ];
+
+    let start_by_other = other_labels.iter().map(|other_label| (other_label.2.to_string(), APTOS_EXECUTOR_OTHER_TIMERS_SECONDS.with_label_values(&[other_label.2]).get_sample_sum())).collect::<HashMap<_, _>>();
```

Comment:
nit: I hate to access tuple fields by `.n` and prefer giving them names by assignment. And looks like `cargo fmt` is needed.

Reply:
updated. forgot this works in lambdas in rust